themes/bootstrap3/templates/Helpers/email-form-fields.phtml

```diff
@@ -40,7 +40,7 @@
 </div>
 <?php $emailFormScript = <<<'JS'
     $('#email_to, #email_from, #email_subject').on('invalid', function($e) {
-        $('#modal .fa-spinner').remove();
+        $('.modal-loading-overlay,.loading-spinner', '#modal').remove();
```

Comment:
I'm also not sure where this applies -- at what point is the spinner added that this is removing?

Reply:
When you submit a form in the lightbox, it switches to loading. If the form is invalid, this ensures that the loading indicators are cleared.

Reply:
My question, though, is if the form is invalid, how does it ever get submitted in the first place? This event seems to trigger when you enter an invalid email address in the form, which prevents the form from submitting... so there's never a loading icon.

Reply:
Yeah, you know I looked closer at both of these cases and they don't apply. I think it might be a mix of out of date and referencing a customization that wasn't contributed.